HelpSource/Classes/SimpleNumber.schelp

```diff
@@ -974,52 +974,85 @@ stores this on the given stream
 method:: printOn
 prints this on the given stream
 
+
 subsection:: random
+Unless noted otherwise, when the receiver and arguments to these methods are all link::Classes/Integer::s, the return will be as well.
 
 method:: coin
 Let emphasis::x:: be the receiver clipped to the range [0, 1]. With probability emphasis::x::, return true. With probability 1 - emphasis::x::, return false.
+code::
+0.4.coin;
+
+{ SinOsc.ar(110).coin }.plot;
+::
 
 method:: rand
-returns:: Random number from zero up to the receiver, exclusive.
+Random number uniformly distributed in the range [0, this).
+code::
+10.rand;
+
+{ SinOsc.ar(110).rand }.plot;
+::
 
 method:: rand2
-returns:: a random number from -this to +this.
+Random number uniformly distributed in the range [-this, +this].
+code::
+10.rand2;
+
+{ SinOsc.ar(110).rand2 }.plot;
```

Comment:
This is an example of `rand2` for UGen. So this is potentially confusing, especially because the UGen behaves like a sequence of floats, not ints. 

Maybe better:

`{ 10.rand2 }.dup(30).plot.plotMode_(\steps)`

`{ 10.0.rand2 }.dup(30).plot.plotMode_(\steps)`

There are more such cases, maybe you can reconsider them?

Edit: I just saw that you copy the methods into operators. This is good, but on the other hand, there we may want to collect a more varied set of examples. It is a trade-off in documentation.

I'd say, keep this one really short and make a more extended one for Operators.

Reply:
i'm not terribly interested in authoring a collection of good examples; i believe my change is an improvement to the documentation even if i deleted most of the existing ones (which are... mediocre).

i'll add a code block before the copymethods in the Operators helpfile to indicate these operators work with a diversity of receivers (SimpleNumber, UGen, Array), along with your plot example, and i'll delete the copypasta examples from SimpleNumber.

Reply:
well, i guess the mere inclusion of Array as a receiver is an improvement in its own right.